### PR TITLE
Timestomp

### PR DESCRIPTION
**Pull request by fooris**
@berggren 

**draft implementation for naive timestomp analyzer**

### CODE REVIEW

**Comment by fooris** [comment]
Still needs tests

**Comment by kiddinn** [comment]
Is this ready for a review?

**Comment by fooris** [comment]
I added some tests, I think it is ready for review now

File: timesketch/lib/analyzers/timestomp.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        self.std_info_timestamp = std_info_timestamp
+        if file_names:
+            self.file_names = file_names
+        else:
+            self.file_names = []
+
+class TimestompSketchPlugin(interface.BaseSketchAnalyzer):
+    """Sketch analyzer for Timestomp."""
+
+    NAME = 'timestomp'
```
Comment (berggren) [inline timesketch/lib/analyzers/timestomp.py]: Can we refactor this to be named "ntfs_timestomp" instead?

**Comment by berggren** [comment]
@fooris PTAL

**Comment by kiddinn** [comment]
gentle ping, what is the status of this?

**Comment by fooris** [comment]
Hi, sorry have been very busy. Will try to get it done this weekend.

**Comment by fooris** [comment]
@googlebot I signed it!

**Comment by fooris** [comment]
> Looks good, some comments. @fooris PTAL

Done. :-)

**Pull request merged**